packages/rn-tester/js/components/ListExampleShared.js

```diff
@@ -359,12 +359,18 @@ const styles = StyleSheet.create({
   },
   // [TODO(macOS GH#774)
   selectedItem: {
-    backgroundColor: PlatformColor('selectedContentBackgroundColor'),
+    backgroundColor: Platform.select({
+      macos: PlatformColor('selectedContentBackgroundColor'),
+      default: 'blue',
```

Comment:
Turns out RN iOS is missing the PlatformColor [tintColor](https://developer.apple.com/documentation/uikit/uiview/1622467-tintcolor) 

May file a bug later.. that one feels important

Reply:
https://github.com/microsoft/react-native-macos/issues/1388